tests/foreman/cli/test_host.py

```diff
@@ -2558,3 +2559,65 @@ def test_positive_dump_enc_yaml(default_sat):
     assert f'fqdn: {default_sat.hostname}' in enc_dump
     assert f'ip: {default_sat.ip_addr}' in enc_dump
     assert 'ssh-rsa' in enc_dump
+
+
+# -------------------------- HOST TRACE SUBCOMMAND SCENARIOS -------------------------
+def generate_custom_rhel_repo_string(major, minor):
+    domain = 'http://download.eng.bos.redhat.com'
```

Comment:
Remove this immediately as its pointing to internal / private repository and make this part of dynaConf from where u ll access this URL from confs.

Let me know if u need any help to understand this !